pkg/collector/corechecks/snmp/config_test.go

```diff
@@ -749,19 +749,10 @@ func Test_snmpConfig_refreshWithProfile(t *testing.T) {
 	assert.Equal(t, oidConfig{
 		scalarOids: []string{
 			"1.2.3.4.5",
-			"1.3.6.1.2.1.1.5.0",
-			"1.3.6.1.2.1.1.1.0",
-			"1.3.6.1.2.1.1.2.0",
 		},
 		columnOids: []string{
 			"1.2.3.4.6",
 			"1.2.3.4.7",
-			"1.3.6.1.2.1.31.1.1.1.1",
-			"1.3.6.1.2.1.31.1.1.1.18",
-			"1.3.6.1.2.1.2.2.1.2",
-			"1.3.6.1.2.1.2.2.1.6",
-			"1.3.6.1.2.1.2.2.1.7",
-			"1.3.6.1.2.1.2.2.1.8",
 		},
```

Comment:
Note for reviewers: removed since those OIDs are related to Device Metadata collection and added at build config step.